solidity/contracts/converter/BancorConverter.sol

```diff
@@ -986,15 +1013,28 @@ contract BancorConverter is IBancorConverter, TokenHandler, SmartTokenController
         return y;
     }
 
+    /**
+      * @dev calculates the nearest integer to a given quotient
+      * 
+      * @param _n   quotient numerator
+      * @param _d   quotient denominator
+      * @return the nearest integer to the given quotient
+    */
     function roundDiv(uint256 _n, uint256 _d) public pure returns (uint256) {
         return (_n + _d / 2) / _d;
     }
 
+    /**
+      * @dev calculates the average number of decimal digits in a given list of values
+      * 
+      * @param _values  list of values (each of which assumed positive)
+      * @return the average number of decimal digits in the given list of values
+    */
     function geometricMean(uint256[] memory _values) public pure returns (uint256) {

```

Comment:
same for this one - generic math function

Reply:
@yudilevi : Same as above.